### PR TITLE
fix: Builds are failing on Dependabot PRs

### LINKED ISSUE #229: Workflow error when using action-hosting-deploy
`Error: Input required and not supplied: firebaseServiceAccount`

Opened issue: https://github.com/FirebaseExtended/action-hosting-deploy/issues/134
Edit: Found the cause so closed the issue, see comment below.

### PR DESCRIPTION
**Pull request by cobraz**
Potential alternate fix for issues described in #229 and [dependabot/dependabot-core#3253](https://github.com/dependabot/dependabot-core/issues/3253#issuecomment-852541544).

Instead of moving to a sub-optimal workflow, I suggest we move away from Dependabot being able to deploy stuff. I'd rather deploy if need be, not have it done in normal operation.

fixes #229
closes #248

### CODE REVIEW

File: .github/workflows/pull-request.yml
Comment (cobraz) [inline .github/workflows/pull-request.yml:65]: Notice that I even added Lighthouse-testing. Not only because we don't have a public URL, but also because it seems _to much_ to run Lighthouse tests every dependency bump.

**Comment by ws333** [comment]
@cobraz Nice solution! But just to make sure I understand this correctly. Will this disable the preview? If so we would need a workflow to trigger manually if we would like to check a preview before merging.

**Comment by cobraz** [comment]
Yes, this will disable the preview. If we want (and need), we can create a separate workflow that builds and creates a preview either when we add a label or a slash command. (Probably only when users with write access to this repository ask for it)

**Comment by ws333** [comment]
> Yes, this will disable the preview. If we want (and need), we can create a separate workflow that builds and creates a preview either when we add a label or a slash command. (Probably only when users with write access to this repository ask for it).
> 
> This only applies to Dependabot, so normal pull requests will work fine :)

Ok :) To me a slash command e.g. `/preview` would be helpful in cases where I'm not sure how the merge will affect the website. E.g. when there is a new major version release of a package.

**Comment by cobraz** [comment]
> > Yes, this will disable the preview. If we want (and need), we can create a separate workflow that builds and creates a preview either when we add a label or a slash command. (Probably only when users with write access to this repository ask for it).
> > This only applies to Dependabot, so normal pull requests will work fine :)
> 
> Ok :) To me a slash command e.g. `/preview` would be helpful in cases where I'm not sure how the merge will affect the website. E.g. when there is a new major version release of a package.

I agree, let's look into that at a later date. Is this ready to merge?

**Review by ws333 — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**
